packages/website/src/pages/foundations/Iconography.tsx

```diff
@@ -1,61 +1,181 @@
-import {ITableHOCProps, TableHOC, tableWithFilter, TableWithFilterProps} from '@coveord/plasma-react';
 import * as PlasmaReactIcons from '@coveord/plasma-react-icons';
-import {HTMLAttributes, ComponentType, FunctionComponent} from 'react';
 import IconographyDemo from '@examples/foundations/Iconography/Iconography.demo?demo';
+import {Fragment, FunctionComponent} from 'react';
 
+import {
+    Box,
+    Card,
+    Center,
+    Collapse,
+    ColumnDef,
+    CopyToClipboard,
+    createColumnHelper,
+    FilterFn,
+    getFilteredRowModel,
+    Grid,
+    Group,
+    renderTableCell,
+    Row,
+    SimpleGrid,
+    Space,
+    Table,
+    TableLayout,
+    TableLayoutProps,
+    TableProps,
+    Title,
+    useDisclosure,
+    useTable,
+} from '@coveord/plasma-mantine';
+import {rankItem} from '@tanstack/match-sorter-utils';
 import {PageLayout} from '../../building-blocs/PageLayout';
 
 const {iconsList, ...Icons} = PlasmaReactIcons;
 
-const TableWithFilter: ComponentType<
-    React.PropsWithChildren<TableWithFilterProps & ITableHOCProps & HTMLAttributes<HTMLTableElement>>
-> = tableWithFilter()(TableHOC);
+type IconSet = (typeof iconsList)[number];
 
-type IconSet = {iconName: string; variants: Array<keyof typeof Icons>};
+const IconSetCard = <T,>({getVisibleCells}: Row<T>) => {
+    const [opened, {toggle}] = useDisclosure(false);
+    const cells = getVisibleCells();
+    const nameCell = cells.find(({column}) => column.id === 'name');
+    const iconCell = cells.find(({column}) => column.id === 'render');
+    const variantsCell = cells.find(({column}) => column.id === 'variants');
+    return (
+        <div>
```

Comment:
Why this div wrapping the card ?

Reply:
Without it when expanding one card, the two others would stretch as well to match the height of the expanded one. With the `div` its the empty space that stretches.
<img width="1245" alt="image" src="https://github.com/coveo/plasma/assets/35579930/932f06d7-c389-47a8-97bb-0911fae77cd9">